pkg/cmd/component/cmd/ui/ui.go

```diff
@@ -403,3 +403,40 @@ func AddEnvironmentName() string {
 	handleError(err)
 	return strings.TrimSpace(environmentName)
 }
+func ApplicationOutputPath() string {
+	var path string
+	prompt := &survey.Input{
+		Message: "Provide a Path to where the Application is located",
+		Help:    "This is the path where application configuration is stored locally before you push it to the repository GitRepoURL",
+	}
+	err := survey.AskOne(prompt, &path, validateAppPath())
+	handleError(err)
+	return strings.TrimSpace(path)
+}
+
+func validateAppPath() survey.Validator {
+	return func(input interface{}) error {
+		return validateAppFolder(input)
+	}
+}
+
+func validateAppFolder(input interface{}) error {
+	if u, ok := input.(string); ok {
```

Comment:
I think you have to revisit this validation of the different scenarios.  We want to be able to support `kam init` from nothing as well.

Reply:
I have made the changes for that in my latest commit.